### ISSUE TITLE
Images with no annotations

### ISSUE THREAD

**Issue by lgvaz** [lgvaz/mantisshrimp#25]
The torchvision model will throw an error if images with no annotations are passed to it. I think we need to preemptively  remove these images.

Note that a transform (like random crop or zoom) can remove the items from an image (potentially leaving it with no objects).

Related to [this](https://discuss.pytorch.org/t/torchvision-faster-rcnn-empty-training-images/46935/12) and [this](https://github.com/pytorch/vision/issues/1128#issuecomment-512229230)

**Comment by lgvaz** [comment]
A solution discussed in the forum given by the link is to create a fake bounding box with label 0 (background) when no object is present. Quoting the forum:
"I set the values to be (0,1,2,3) for label “0”."

**Comment by lgvaz** [comment]
And the images with no annotations **should** already by removed by the `DataParser`, so this problem only happens if a transform removes all objects from the image
